### PR TITLE
Release V1.6.0

### PR DESCRIPTION
**Pull request by omenUchiha**
## Description

**Features**
- Anonymous Post
- Hide Post
- Reported Bugs
- Post, Like & Comment Variables

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Hotfix (fixes a breaking changes)
- [x] New Feature (non-breaking change which adds functionality)
- [ ] Other, please describe:

### Author Checklist

- [x] Assignee added
- [x] Reviewer added
- [x] Description added
- [ ] Unit test cases added

### Reviewer Checklist

- [ ] PR title starts with Jira ticket number prefix
- [ ] If hotfix, Add Hotfix as prefix in PR title
- [ ] Label added
- [ ] Commit messages follows standard format (added, changed, removed, code commented and chore)
- [ ] Description about the Feature/ Bugs added

### CODE REVIEW

File: core/package.json [hunk 1]
```diff
@@ -20,13 +20,15 @@
...
   },
   "dependencies": {
+    "@ast-grep/napi-darwin-x64": "^0.28.1",
     "@aws-sdk/client-s3": "3.556.0",
     "@aws-sdk/credential-provider-cognito-identity": "3.556.0",
     "@aws-sdk/credential-providers": "3.556.0",
     "@emotion/react": "11.11.3",
     "@emotion/styled": "11.11.0",
-    "@likeminds.community/feed-js": "^1.13.0",
+    "@likeminds.community/feed-js": "^1.14.0",
```
Comment (jaixchaudhary) [inline core/package.json:29]: remove caret

File: core/package.json [hunk 2]
```diff
@@ -98,6 +101,7 @@
     "eslint-plugin-react-refresh": "^0.4.5",
     "husky": "^9.0.11",
     "node-sass": "^9.0.0",
+    "pluralize": "^8.0.0",
```
Comment (jaixchaudhary) [inline core/package.json:104]: remove pluralize

File: core/package.json (hunk 1 above)
Comment (omenUchiha) [reply core/package.json:29]: done

File: core/src/components/LMFeedCreatePost.tsx [hunk 3]
```diff
@@ -224,7 +223,8 @@ const LMFeedCreatePost = ({
                       setOpenCreatePostDialog(!openCreatePostDialog);
                     }}
                   >
-                    {POST}
+                    {/* {changePostCase(WordAction.FIRST_LETTER_CAPITAL_SINGULAR)} */}
```
Comment (jaixchaudhary) [inline core/src/components/LMFeedCreatePost.tsx:226]: remove unnecessary comments

File: core/src/hooks/useLMFetchFeeds.ts [hunk 4]
```diff
@@ -25,6 +25,7 @@ import { FeedListActionsAndDataStore } from "../shared/types/cutomCallbacks/data
 import LMFeedUserProviderContext from "../contexts/LMFeedUserProviderContext";
 
 import { ComponentDelegatorListener } from "../shared/types/cutomCallbacks/callbacks";
+import { getDisplayMessage } from "../shared/utils";
 
 // import { GetPinPostResponse } from "../shared/types/api-responses/getPinPostResponse";
```
Comment (jaixchaudhary) [inline core/src/hooks/useLMFetchFeeds.ts:30]: remove unnecessary comments

File: core/package.json (hunk 2 above)
Comment (omenUchiha) [reply core/package.json:104]: done

File: core/src/hooks/useLMTopicDropdown.ts [hunk 5]
```diff
@@ -17,7 +17,10 @@ import {
 import { ComponentDelegatorListener } from "../shared/types/cutomCallbacks/callbacks";
 export function useTopicDropdown(
   currentSelectedTopicIds?: string[],
-  // setCurrentSelectedTopicIds?: React.Dispatch<string[]>,
+  // setCurrentSelectedTopicIds?: React.Dispatch<
+  //   React.SetStateAction<string[]> | string[]
+  // >,
+
```
Comment (jaixchaudhary) [inline core/src/hooks/useLMTopicDropdown.ts:23]: remove unnecessary comments

File: core/src/hooks/useLMUserProvider.ts [hunk 6]
```diff
@@ -7,10 +7,13 @@ import { LMClient } from "../shared/types/dataLayerExportsTypes";
...
 import {
   InitiateUserRequest,
+  LMResponseType,
   ValidateUserRequest,
 } from "@likeminds.community/feed-js";
 import { LMFeedCustomEvents } from "../shared/customEvents";
 import { LMFeedCustomActionEvents } from "../shared/constants/lmFeedCustomEventNames";
+import { GetCommunityConfigurationsResponse } from "@likeminds.community/feed-js/dist/initiateUser/model/GetCommunityConfigurationsResponse";
+import { TokenValues } from "../shared/enums/tokens";
 // import { TokenValues } from "../shared/enums/tokens";
```
Comment (jaixchaudhary) [inline core/src/hooks/useLMUserProvider.ts:17]: remove unnecessary comments

File: core/src/hooks/useLMFetchFeeds.ts (hunk 4 above)
Comment (omenUchiha) [reply core/src/hooks/useLMFetchFeeds.ts:30]: done

File: core/src/components/LMFeedCreatePost.tsx (hunk 3 above)
Comment (omenUchiha) [reply core/src/components/LMFeedCreatePost.tsx:226]: done

File: core/src/hooks/useLMTopicDropdown.ts (hunk 5 above)
Comment (omenUchiha) [reply core/src/hooks/useLMTopicDropdown.ts:23]: done

File: core/src/shared/utils.ts [hunk 7]
```diff
@@ -1,7 +1,14 @@
...
 import relativeTime from "dayjs/plugin/relativeTime";
 import dayjs from "dayjs";
-// import { AvatarProps } from "./types/models/avatarProps";
+
+import { Pluralize } from "./variables";
+
+import { TokenValues } from "./enums/tokens";
+import { WordAction } from "./enums/wordAction";
+import { LMDisplayMessages } from "..";
+// import { GetCommunityConfigurationsResponse } from "@likeminds.community/feed-js/dist/initiateUser/model/GetCommunityConfigurationsResponse";
```
Comment (jaixchaudhary) [inline core/src/shared/utils.ts:10]: remove unnecessary comments

File: core/src/hooks/useLMUserProvider.ts (hunk 6 above)
Comment (omenUchiha) [reply core/src/hooks/useLMUserProvider.ts:17]: done

**Review by jaixchaudhary — APPROVED** [review]

**Pull request merged**